### DISCUSSION TITLE
在windows环境下如何配置和使用httprunner4.1。

### DISCUSSION THREAD

**Discussion by kinc123** [httprunner/httprunner#1327] [3 - 分享（Share）]
使用httprunner4.0版本时踩了一些坑，特此分享。
安装部署：
windows环境下执行部署脚本经常失败，无法匹配到zip格式的文件，建议直接到官网进行下载。

下载地址：https://github.com/httprunner/httprunner/releases
下载后的windows环境下的tar.gz文件手动改为zip格式并解压。
a）在C盘根目录下创建tmp/hrp目录（自定义目录），将hrp.exe文件放在目录下。
b)  配置hrp服务的环境变量，我的电脑——属性——高级系统设置——环境变量，在PATH下新增系统变量，将上一步的绝对路径写入PATH。

**Comment by debugtalk** [comment]
@kinc123 感谢反馈，这里是一个 bug。

经确认，在 Windows 环境编译生成的包为 `.tar.gz` 格式，而在解压的时候，针对 Windows 是采用 unzip 解压 `.zip` 格式，两者不匹配。

我已经在 v4.1.2 中修复了，做法是统一使用 tar 来解压 `.tar.gz` 格式。

#1336
